### PR TITLE
Feature/email template deactivated account

### PR DESCRIPTION
**Pull request by peggles2**
Add email template for warning users that their account is going to be de-activated.

### CODE REVIEW

File: spec/mailers/emailer_spec.rb [hunk 1]
```diff
@@ -6,6 +6,19 @@
   include EmailSpec::Matchers
   fixtures :affiliates, :users, :features, :memberships
 
+  describe '#account_deactivation_warning' do
```
Comment (MothOnMars) [inline spec/mailers/emailer_spec.rb]: Class methods should be prepended with a period: `describe '.account_deactivation_warning'` (I see the other class methods in this spec are prepended with a `#`, but that ain't right)
http://www.betterspecs.org/#describe

File: spec/mailers/emailer_spec.rb [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+    let(:user) { users(:not_active_76_days) }
+    let(:message) do
+      'automatically deactivate your account in the next 14 days due to inactivity.'
+    end
+
+    subject(:deactivate_email) { Emailer.account_deactivation_warning(user, 76.days.ago.to_date) }
```
Comment (MothOnMars) [inline spec/mailers/emailer_spec.rb]: When describing a method, it is best to use a subject name consistent with the method being tested:
```ruby
subject(:account_deactivation_warning) do
  Emailer.account_deactivation_warning(user, 76.days.ago.to_date)
end
```

File: spec/mailers/emailer_spec.rb (hunk 1 above)
Comment (peggles2) [reply spec/mailers/emailer_spec.rb]: ok I was trying to just be consistent with what is currently there.

**Comment by MothOnMars** [comment]
LGTM

**Pull request merged**